app/views/components/_translation-nav.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="app-c-translation-nav">
+  <ul class="app-c-translation-nav__list">
+    <% translations.each.with_index do |translation, i| %>
+      <li class="app-c-translation-nav__list__language<%= ' last' if i == translations.length-1 %>">
+        <% if translation[:active].present? %>
```

Comment:
`translation[:active]` should be a boolean, so we can simply do: `if translation[:active]`